bip-0155.mediawiki

```diff
@@ -134,7 +134,7 @@ See the appendices for the address encodings to be used for the various networks
 
 Introduce a new message type <code>sendaddrv2</code>. Sending such a message indicates that a node can understand and prefers to receive <code>addrv2</code> messages instead of <code>addr</code> messages. I.e. "Send me addrv2".
 
-<code>sendaddrv2</code> SHOULD be sent after receiving the <code>verack</code> message from the peer.
+<code>sendaddrv2</code> SHOULD be sent before sending the <code>verack</code> message to the peer.
```

Comment:
Perhaps pedantic, but for the sake of precision, should this state
```suggestion
<code>sendaddrv2</code> SHOULD be sent after receiving the <code>version</code> from the peer and before sending the <code>verack</code> message to the peer.
```

Reply:
To compare with BIP 339: "The wtxidrelay message MUST be sent in response to a version message from a peer whose protocol version is >= 70016 and prior to sending a verack. A wtxidrelay message received after a verack message MUST be ignored or treated as invalid."

Reply:
I think wtxidrelay had to be stricter in enforcing that this happened before the verack. addrv2 doesn't have to be this strict (at least with the implementation as is now) and can leave this up to the implementation, see https://github.com/bitcoin/bitcoin/pull/20564#discussion_r537730772

Reply:
Changed to `MUST` and reworded to mimic BIP-339 / wtxidrelay.

@jonatack, thanks for the BIP-339 quote!